src/interface.rs

```diff
@@ -198,7 +201,7 @@ impl Interface {
         running_state: &RunningState,
     ) -> Result<()> {
         let time_since_start = timer::time_since_start(context);
-        let screen_size = graphics::drawable_size(context);
+        let screen_size = self.screen_size;
```

Comment:
I don't think we need to keep `let screen_size = self.screen_size` here. We can find where we are using it and replace with `self.screen_size`

Reply:
You mentioned that we need to clone here for using in a closure later. In that case we're good to go!